### PR TITLE
Resources of the ASM MCP Multi Clusters Workshop

### PR DESCRIPTION
**Pull request by mathieu-benoit**
Make the resources of the ASM MCP Multi Clusters Workshop publicly available for students, customers and Googlers.

- Took as-is the `terraform` and `instructions` folder from the Qwiklabs repo
- Created the `README.md` which may need a review
- Places these resources under `demos` (not `docs`)

TODOs (after merge in `main`):
- [ ] Add the copyright header for the resources in the `terraform` folder yet, something to do later if possible/needed

### CODE REVIEW

**Comment by ckim328** [comment]
sweet - we should add a reminder to add the workshop link once it works

File: demos/asm-mcp-multi-clusters/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Resources used during IstioCon 2022 for the Anthos Service Mesh (ASM) workshop
+
+The pre-provisioned infrastructure of this workshop is in the [terraform folder](./terraform).
+
+The actual instructions of the workshop could be found [here](./instructions/en.md).
+
+Here is the architecture diagram for this setup:
+![Architecture diagram](./instructions/img/architecture.png)
+
```
Comment (ckim328) [inline demos/asm-mcp-multi-clusters/README.md:9]: could we also mention and have a link for BoA

File: demos/asm-mcp-multi-clusters/terraform/variables.tf [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  description = "The GCP region to apply this config to."
+}
+variable "gcp_zone" {
+  type        = string
+  description = "The GCP zone to apply this config to."
+}
+variable "gcp_user_id" {
+  type        = string
+  description = "User Id"
+  default     = "mikegcoleman@cloudadvocacy.joonix.net"
```
Comment (ckim328) [inline demos/asm-mcp-multi-clusters/terraform/variables.tf:16]: remove this

**Review by ckim328 — CHANGES_REQUESTED** [review]

File: demos/asm-mcp-multi-clusters/terraform/variables.tf (hunk 2 above)
Comment (mathieu-benoit) [reply demos/asm-mcp-multi-clusters/terraform/variables.tf]: Good catch, done.

File: demos/asm-mcp-multi-clusters/README.md (hunk 1 above)
Comment (mathieu-benoit) [reply demos/asm-mcp-multi-clusters/README.md:10]: Good idea, done.

**Review by ckim328 — APPROVED** [review]
LGTM!

**Review by NimJay — APPROVED** [review]
Thanks for doing this, @mathieu-benoit!
Approved.

Should we also add a link (in the README.md) to the [actual QwikLab](https://googlesolutions.qwiklabs.com/classrooms/13051/labs/120434)?

**Pull request merged**